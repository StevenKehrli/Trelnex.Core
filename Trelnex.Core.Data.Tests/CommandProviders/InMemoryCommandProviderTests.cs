namespace Trelnex.Core.Data.Tests.CommandProviders;

public class InMemoryCommandProviderTests : CommandProviderTests
{
    [SetUp]
    public async Task TestFixtureSetup()
    {
        // create our command provider
        var factory = await InMemoryCommandProviderFactory.Create();

        _commandProvider =
            factory.Create<ITestItem, TestItem>(
                typeName: "test-item",
                TestItem.Validator,
                CommandOperations.All);
    }
}
