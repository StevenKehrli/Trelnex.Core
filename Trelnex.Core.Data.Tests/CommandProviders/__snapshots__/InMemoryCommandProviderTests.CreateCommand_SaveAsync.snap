{
  "Item": {
    "Id": "2a4cb3ec-6624-4fc6-abc4-6a5db019f8f9",
    "PartitionKey": "b297ff5b-2ab5-4b8d-9dfd-57d2e1d8c3d2",
    "TypeName": "test-item",
    "CreatedDate": "2025-01-07T00:54:42.008023Z",
    "UpdatedDate": "2025-01-07T00:54:42.008023Z",
    "DeletedDate": null,
    "IsDeleted": null,
    "ETag": "3135d648-065d-49e6-9450-d5187b215232",
    "Message": "Message #1"
  }
}
